visualization/package.json

```
@@ -120,36 +115,24 @@
 		"@types/three": "^0.126.0",
 		"@typescript-eslint/eslint-plugin": "^5.44.0",
 		"@typescript-eslint/parser": "^5.44.0",
-		"babel-loader": "^9.1.2",
 		"bestzip": "^2.2.0",
-		"clean-webpack-plugin": "^4.0.0",
-		"css-loader": "^6.0.0",
 		"eslint": "^8.28.0",
 		"eslint-config-prettier": "^8.5.0",
 		"eslint-plugin-unicorn": "^45.0.0",
 		"eslint-plugin-unused-imports": "^2.0.0",
-		"file-loader": "^6.2.0",
 		"html-loader": "^4.0.0",
-		"html-loader-jest": "^0.2.1",
-		"html-webpack-plugin": "^5.3.1",
-		"jest": "27.x.x",
-		"jest-preset-angular": "^9.0.7",
+		"jest": "28.1.3",
```

```suggestion
		"jest": "^28.1.3",
```

Updating to v29 directly probably requires further work?

---

Maybe? Lets target this in another PR :)